### ISSUE TITLE
Won't install on SQL Server Mgmt Studio 2016

### ISSUE THREAD

**Issue by robertdnewman** [nycdotnet/TSqlFlex#49]
I have SSMS 2008,2012,2014 and 2016 on my workstation.
After installing Flex, it comes up in all the versions except 2016.
Any thoughts?

**Comment by nycdotnet** [comment]
Hi,
Yup.  SSMS 2016 changed the model for plugins so it's not going to work for now. I haven't investigated the new model yet so unfortunately there is no schedule yet for this support.  I haven't heard from Redgate officially either of they are going to change the SIP framework to work with 2016 either.  So please stay tuned.  ;)

**Comment by robertdnewman** [comment]
Thanks!

**Comment by gregbandoian** [comment]
Is this still an ongoing issue?  I'm unable to install it on SSMS 2017.  Thanks!

**Comment by nycdotnet** [comment]
Yes.  I have gotten T-SQL Flex to work in Visual Studio 2017 (which shares the SSMS 2017 component model) and I might work around the integration issue by just writing my own connection picker.

**Comment by robertdnewman** [comment]
Great news!



*From:* Steve Ognibene <notifications@github.com>
*Sent:* Thursday, April 05, 2018 5:58 PM
*To:* nycdotnet/TSqlFlex <TSqlFlex@noreply.github.com>
*Cc:* robertdnewman <rnewman@onebox.com>; Author <author@noreply.github.com>
*Subject:* Re: [nycdotnet/TSqlFlex] Won't install on SQL Server Mgmt Studio
2016 (#49)



Yes. I have gotten T-SQL Flex to work in Visual Studio 2017 (which shares
the SSMS 2017 component model) and I might work around the integration
issue by just writing my own connection picker.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/nycdotnet/TSqlFlex/issues/49#issuecomment-379119012>,
or mute the thread
